### ISSUE TITLE
Tests for np.diff implementation missing

### ISSUE THREAD

**Issue by ayush-1506** [google/jax#3505]
np.diff is implemented here: https://github.com/google/jax/blob/master/jax/numpy/lax_numpy.py#L967, but I couldn't find any tests for it.

**Comment by mattjj** [comment]
Thanks for asking! Using GitHub's "blame" tab, I found it was added in #588, and I believe [this line runs tests for it](https://github.com/google/jax/blob/3fff83790742a1b0a468032ecff6f8809fde57d9/tests/lax_numpy_test.py#L261).

One quick way to "test for tests" is to add an `assert False` to the implementation of a function, and see what tests (if any) fail!

I think `jnp.diff` has test coverage, so I'm going to close this issue.

**Comment by clemisch** [comment]
Nitpick: `diff` is tested, but IIUC the `axis` parameter is not varied?

**Comment by hawkinsp** [comment]
We welcome PRs!
